spec/aws_loader_spec.rb

```diff
@@ -0,0 +1,25 @@
+RSpec.describe AwsLoader do
```

Comment:
Add the class `ModuleTest` as a dummy class here above this line.
The ModuleTest includes the `aws_loader_spec` module